demos/home-automation/rust/Cargo.toml

```
@@ -21,6 +21,7 @@ name = "home_automation_lib"
 [dependencies]
 slint = { path = "../../../api/rs/slint" }
 chrono = "0.4"
+kira = "0.10"
```

To reduce build times, I suggest to include only the audio format needed.

```suggestion
kira = { version = "0.10", default-features = false, features = ["wav"] }
```